SharedUtils/src/main/java/com/braintreepayments/api/sharedutils/SynchronousHttpClient.kt

```diff
@@ -0,0 +1,66 @@
+package com.braintreepayments.api.sharedutils
+
+import androidx.annotation.RestrictTo
+import java.net.HttpURLConnection
+import javax.net.ssl.HttpsURLConnection
+import javax.net.ssl.SSLSocketFactory
+
+/**
+ * This class performs an http request on the calling thread. The external caller is
+ * responsible for thread scheduling to ensure that this is not called on the main thread.
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+internal class SynchronousHttpClient(
+    private val socketFactory: SSLSocketFactory,
+    private val parser: HttpResponseParser
+) {
+
```

Comment:
```suggestion
```

Reply:
I feel that the new line here helps with readability, adding some separation from the class's constructor and the contents of the class.